cmake/configs/qurt_eagle_release.cmake

```diff
@@ -70,7 +70,7 @@ set(config_module_list
 	modules/uORB
 	modules/commander
 	modules/controllib
-
+	
```

Comment:
Tried to remove this change, but couldn't.  Is there a git command to move a particular file to a previous commit?  I know the command to revert an entire commit.